plutus-core/untyped-plutus-core/test/Spec.hs

```diff
@@ -53,11 +57,11 @@ test_bigInteger = testCase "big integer" $ do
         justUnder :: Integer
         justUnder = 2 ^ (55 * 8 :: Integer) - 1
```

Comment:
I wonder what `size` flat reports for all these justunder/justover values and if it is worth to add them to these assertions for regression testing

Reply:
y the `Flat.size`. Shouldn't this be around the 64bytes? I don't remember if `size` returns bits or bytes though